source/core/parameters_lagrangian.cc

```diff
@@ -296,28 +296,43 @@ namespace Parameters
                           "1",
                           Patterns::Integer(),
                           "Insertion frequency");
-        prm.declare_entry("insertion direction sequence",
-                          "0,1,2",
-                          Patterns::List(Patterns::Integer(), 2, 3),
-                          "First direction of particle insertion");
+
+        // Clearing box:
         prm.declare_entry(
-          "insertion box points coordinates",
+          "clearing particles",
+          "false",
+          Patterns::Bool(),
+          "State whether particles should be cleared on insertion.");
+
+        prm.declare_entry(
+          "clearing box points coordinates",
           "0. , 0. , 0. : 1. , 1. , 1.",
           Patterns::List(
             Patterns::List(Patterns::Double(), 2, 3, ","), 2, 2, ":"),
```

Comment:
I feel like the natural separator is ",". I think the default separator for the file list should either be a comma or whitespace. I would use commas (csv), to follow the patter of the parameters, kinda. The semicolon is kinda weird.

Reply:
Regardless of your separator, just don't forget to mention it in the documentation :)